jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/JettyRunMojo.java

```diff
@@ -285,7 +286,15 @@ public void configureWebApplication() throws Exception
        if (useTestScope && (testClassesDirectory != null))
            webApp.setTestClasses (testClassesDirectory);
 
-       webApp.setWebInfLib(getDependencyFiles());
+        MavenProjectHelper mavenProjectHelper = new MavenProjectHelper(project);
+        List<File> webInfLibs = getWebInfLibArtifacts(project).stream()
+                .map(a -> {
+                    Path p = mavenProjectHelper.getArtifactPath(a);
+                    getLog().debug("Artifact " + a.getId() + " loaded from " + p + " added to WEB-INF/lib");
+                    return p.toFile();
+                }).collect(Collectors.toList());
+        getLog().info("WEB-INF/lib initialized (at root)");
```

Comment:
sounds more like a debug rather than info as it doesn't provide interested informations.
can you change to debug?

Reply:
done